src/api/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from "express";
+import { UserRoutes } from "./routes/user";
+
+export class Routes {
+  app: Router = Router();
+
+  constructor() {
+    this.init();
+  }
+
+  private init(): void {
+    new UserRoutes(this.app);
```

Comment:
Please also handle errors for non-existing routes, so user see the text message instead of stacktrace, ex.

`GET localhost:3000/nonexistentapi/`

causes

```
Error: Not Found
    at expressApp.use (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\src\loaders\express.loader.ts:30:21)
    at Layer.handle [as handle_request] (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:275:10)
    at jsonParser (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\body-parser\lib\types\json.js:110:7)
    at Layer.handle [as handle_request] (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\Cyprian_Gepfert\projects\epam\mentoring\nodeJS-ihor-voronovskyi\node_modules\express\lib\router\index.js:284:7
```

Reply:
Changed stacktrace to user friendly message.
File express.loader.ts line 30